cartoframes/io/carto.py

```diff
@@ -6,6 +6,17 @@
 from ..core.managers.context_manager import ContextManager
 from ..utils.utils import is_sql_query
 
+__all__ = [
```

Comment:
Why is this added?

Reply:
To import the methods directly, I couldn't without it, but maybe there's another way.

Reply:
Inside the code, we use the full path (cartoframes.io.carto.read_carto). In the public API all these methods are exposed in the top level (cartoframes.read_carto). By default, it should load all these methods without the \_\_all\_\_ variable.

Reply:
Oh, I didn't know it! ok, fixed 👍